docs/dev/libvirt/README.md

```diff
@@ -286,6 +286,38 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### Console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console depends on the route which is created by auth operator.
+To make it works we need to first create the manifests and edit the `domain` for ingress config, before directly creating the cluster.
+
+Following steps are required to have console operator up and working.
+```console
+// Step-1
+// Add another domain entry in the openshift.conf which used by dnsmasq
+// Here `tt.testing` is the domain which I choose when running the installer.
+// Here the IP in the address belong to one of the worker node.
+$ cat /etc/NetworkManager/dnsmasq.d/openshift.conf
+server=/tt.testing/192.168.126.1
+address=/.apps.tt.testing/192.168.126.51
+
+// Step-2
+// Make sure you restart the NetworkManager after change in the openshift.conf
+$ sudo systemctl restart NetworkManager
+
+// Step-3
+// Create the manifests
+$ openshift-install --dir $INSTALL_DIR create manifests
+
+// Step-4
+// Domain entry for cluster-ingress-02-config.yml file should be following (here domain is what your created in Step-1)
+$ cat $INSTALL_DIR/manifests/cluster-ingress-02-config.yml | grep domain
```

Comment:
no need to `cat`, just `grep domain $INSTALL_DIR/manifests/cluster-ingress-02-config.yml`

Reply:
Actually, I suggest turning it into:

```sed -i 's/test1.//g' mycluster/manifests/cluster-ingress-02-config.yml```

Which is what I've been using and it makes things easy for people.

Reply:
@zeenix this will only work if you provided `test1` as your domain name.

Reply:
@praveenkumar Right, it should be:

```
sed -i 's/YOUR_DOMAIN_NAME.//g' mycluster/manifests/cluster-ingress-02-config.yml
```
then. :)

Reply:
@praveenkumar could you please update this or let me know if you disagree?